pages/mi/mi-10-steps.md

```diff
@@ -41,6 +41,9 @@ To participate in the internship, you will need the following hardware:
 
 Follow the instructions on [Creating Your GitHub Profile Page: A Guide to Markdown & Forking Workflow](mi-github-and-markdown.md).
 
+**NOTE**: Only move onto the next step if you have created the pull request for your profile. Make sure it's merged before continuing.
```

Comment:
We are just making sure people won't forget to create pull request before moving on to step 2, so `Make sure it's merged before continuing` would not be necessary.

Also, as we mentioned in one of the notes in step 8, less is often more. Looking at the first steps document overall, I think we'd like to avoid adding another MDWiki alert. Could you remove the alert syntax as well?

Reply:
Fixed